### DISCUSSION TITLE
Announcements

### DISCUSSION THREAD

**Discussion by devmoek** [devmoek/recently-viewed-section-dawn#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by devmoek** [comment]
why not

**Reply by rongjinggao** [reply]
Hello 

Thank you for this 

I have been able to follow through all the way till the step to add the code to jason file

When i tried to paste the copy I have this error all the time 

Unable to update the file

    Invalid JSON: unexpected characters after the JSON document (after ) at line 218, column 1

Any idea why? 

Thank you
Katherine

**Reply by devmoek** [reply]
Hello, Katherine. It's interesting. Can you send me a store url?

**Comment by rongjinggao** [comment]
Hello

Thank you for getting back to me

Sure here is it

[cid:c7a12b84-1c5c-40df-a842-fa56c4598f17]
________________________________

**Reply by devmoek** [reply]
seems like GitHub blocked your link. Could you please contact me by [email](mailto:info@devmoek.com)?
